src/Frame.h

```diff
@@ -183,8 +183,17 @@ namespace openshot
 		/// Display the wave form
 		void DisplayWaveform();
 
-		/// Get magnitude of range of samples (if channel is -1, return average of all channels for that sample)
-		float GetAudioSample(int channel, int sample, int magnitude_range);
+		/// Get highest value within a range of samples
+		/// 
+		/// @param channel Channel number to scan, or -1 for all channels
+		/// @param sample The starting sample for the range
+		/// @param sample_count The length of the sample range
+		/// @returns The maximum sample value within the range of samples on
+		///          the specified channel, or all channels if channel == -1
```

Comment:
I removed all mention of "average"-anything from the doc comment, because no averaging is _ever_ performed by `juce::AudioBuffer::getMagnitude()`. What it _actually_ does is return the largest sample value from within a range of samples. If a channel is specified, it's from among the samples on that channel. If a channel value isn't specified (or is `-1`, in our case) it's from among the samples on _all_ channels in that range. But the value returned is still the **absolute** maximum, without averaging.

Reply:
Each parameter is explicitly documented, as is the return value.